docs/hardware.rst

```diff
@@ -382,6 +383,14 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
+.. _nuc10_recommendation:
+
+Intel 10th-gen NUC
+~~~~~~~~~~~~~~~~~~
+We have tested and can recommend the `Intel NUC10i5FNH <https://ark.intel.com/content/www/us/en/ark/products/189239/intel-nuc-10-performance-kit-nuc10i5fnh.html>`__.
+It provides two storage options: M.2 SSD storage and a 2.5" secondary storage
+option (SSD or HDD).
+
```

Comment:
@rmol If you want to add details re: WiFi hardware removal, this would be a good place to do so.

Reply:
done.